frontend/src/component/photo/cards.vue

```diff
@@ -90,7 +90,7 @@
               <v-icon color="white" class="select-on">lock</v-icon>
             </v-btn>
 
-            <v-btn :ripple="false"
+            <v-btn v-if="selectMode" :ripple="false"
```

Comment:
If you needed to add this: Is there a bug in our stable release at the moment? Visibility should be set via CSS already.

Reply:
Yes, I think there is a bug - the circle was being shown in the new UI, although I have disabled the selection. There might be other ways to hide it as well, but this check is needed IMO.

Reply:
How you can disable selecting pictures completely? AFAIK you can always select pictures...

Reply:
`selectMode` does exactly that - disable selecting pictures.

Reply:
We've used it in a positive sense to always show the button when at least one item is selected. You should always see it on hover / mouse over, even if no item is selected.